pkg/servicecentral/store.go

```diff
@@ -306,6 +306,11 @@ func serviceDataToService(data *ServiceDataRead) (*creator_v1.Service, error) {
 		service.Spec.Metadata.PagerDuty = pagerDutyMetadata
 	}
 
+	ogMetadata, _ := GetOpsGenieAttribute(data)
```

Comment:
shall we log an error?

Reply:
> shall we log an error?
Considering this is optional, Should this be an error or info message?

Reply:
I would say that failure to make a call to Opsgenie integration manager is still an error, so I would log it at the error level. We should react to being unable to fetch Opsgenie integrations rather than completely ignoring them.

@ash2k WDYT?

Reply:
This is not the path of calling OG int manager,  this call will attempt to find any opsgenie attributes within the service object.

We call opsgenie int manager during the `createOrUpdateServiceMetadata` function when building the notifications.

Reply:
> attempt to find any opsgenie attributes within the service object

If attribute not found, there won't be an error, but just a `nil` result AFAICT?
So then the error means that either the attribute format is invalid, or there is some bug in Voyager code, and we should also log with error, I think.

Reply:
Yep you are correct, an error here is when:

1. we found an issue with an Opsgenie attribute
2. we found more than one Opsgenie attribute.